packages/es-components/src/components/controls/radio-buttons/RadioButton.js

```diff
@@ -126,9 +112,6 @@ RadioButton.propTypes = {
   id: PropTypes.string,
   isDisabled: PropTypes.bool,
   inline: PropTypes.bool,
-  onClick: PropTypes.func,
-  value: PropTypes.any,
-  ariaHide: PropTypes.bool,
```

Comment:
Did the api really change to warrant a major version bump? It looks more like these are just getting added to the input by `{...radioProps}`. Or is the version bump due to the deprecation of the `ariaHide` prop?

Reply:
It was for the ariaHide prop going away.